### PR TITLE
Build ‘When I am on-call’ for web UI #2915

### PR DESCRIPTION
**Pull request by maskin25**
# What this PR does

Polish ‘When I am on-call’ feature

## Which issue(s) this PR fixes

[Build ‘When I am on-call’ for web UI #2915](https://github.com/grafana/oncall/issues/2915)

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: grafana-plugin/src/containers/Rotations/SchedulePersonal.tsx [hunk 1]
```diff
@@ -77,14 +77,16 @@ class SchedulePersonal extends Component<SchedulePersonalProps> {
 
     const storeUser = store.userStore.items[userPk];
 
+    console.log('storeUser', storeUser);
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/Rotations/SchedulePersonal.tsx:80]: console.log

File: grafana-plugin/src/containers/ScheduleSlot/ScheduleSlot.tsx [hunk 2]
```diff
@@ -237,13 +238,24 @@ const RegularEvent = (props: RegularEventProps) => {
       {users.map(({ display_name, pk: userPk, swap_request }) => {
         const storeUser = store.userStore.items[userPk];
 
+        const shiftId = event.shift?.pk;
+        const shift = store.scheduleStore.shifts[shiftId];
+
+        /* if (shiftId && !store.scheduleStore.shifts[shiftId]) {
+          store.scheduleStore.updateOncallShift(shiftId);
+        }
+ */
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/ScheduleSlot/ScheduleSlot.tsx:247]: remove?

File: grafana-plugin/src/containers/Rotations/SchedulePersonal.tsx (hunk 1 above)
Comment (maskin25) [reply grafana-plugin/src/containers/Rotations/SchedulePersonal.tsx]: removed

File: grafana-plugin/src/containers/ScheduleSlot/ScheduleSlot.tsx (hunk 2 above)
Comment (maskin25) [reply grafana-plugin/src/containers/ScheduleSlot/ScheduleSlot.tsx]: removed

**Pull request merged**